README.md

```diff
@@ -31,6 +31,7 @@ and exchanges, inspect broker.
     * [Examples](#examples)
         * [Broker info](#broker-info)
         * [Wire-tapping messages](#wire-tapping-messages)
+	    * [Tap all messages published or delivered](#tap-all-messages-published-or-delivered) 
```

Comment:
TOC #protip nice tool I use is https://github.com/thlorenz/doctoc

Reply:
I'm fine with vim and the vim-markdown-plugin ;-)

Reply:
I ❤️ VIM. Nice plugin `:InsertToc`